package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "generator-create-rc-admin",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "json-schema-faker": "^0.5.0-rcv.44",
+    "react": "^18.2.0",
+    "react-admin": "^4.2.0",
+    "yeoman-generator": "^5.6.1"
+  }
+}
```

Comment:
1. Data provider for these APIs https://www.postman.com/collections/713ecec1a44fc1b53f92 
    _Note: CRUD should be working properly_
2. sidebar label and icon
3. Field type manage (Boolean/Email/URL) 
4. Transformations
5. UI changes to React Admin inspired by X-admin
6. Login Component to embed